pkg/api/handlers/types.go

```diff
@@ -347,24 +347,35 @@ func ImageDataToImageInspect(ctx context.Context, l *libpodImage.Image) (*ImageI
 
 }
 
-func LibpodToContainer(l *libpod.Container, infoData []define.InfoData) (*Container, error) {
+func LibpodToContainer(l *libpod.Container, infoData []define.InfoData, sz bool) (*Container, error) {
 	imageId, imageName := l.Image()
 
 	var (
 		err        error
 		sizeRootFs int64
 		sizeRW     int64
 		state      define.ContainerStatus
+		stateStr   string
```

Comment:
Don't define it here.  Just use `stateStr=state.String()` below.

Reply:
Done